TKGame/TKGame.cs

```diff
@@ -114,10 +115,18 @@ protected override void LoadContent()
             Art.LoadContent(Content);
             Music.LoadContent(Content, 0.15f);
 
-            // Manually adding entities at the moment...
+            // Manually add player instance
             EntityManager.Add(Player.Instance);
-            EntityManager.Add(Enemy.DoomguyEnemy.Instance);
-            EntityManager.Add(Item.DiamondSwordItem.Instance);
+
+            // Spawn a knight enemy
+            EnemyFactory knightFactory = new KnightEnemyFactory();
+            Enemy knight = knightFactory.CreateEnemy();
+            EntityManager.Add(knight);
```

Comment:
This (last 3 lines) can probably be made into a single line of code and will need to be moved out probably into the level class eventually since the game class shouldnt be responsible for spawning things